validator/util_test.go

```diff
@@ -181,7 +181,22 @@ func Test_wrapError(t *testing.T) {
 		args    args
 		wantErr bool
 	}{
-		// TODO: Add test cases.
+		{
+			name: "test",
+			args: args{
+				parent: errors.New("test"),
+				new:    errors.New("test"),
+			},
+			wantErr: true,
+		},
+		{
+			name: "",
+			args: args{
+				parent: nil,
+				new:    nil,
+			},
+			wantErr: false,
+		},
```

Comment:
Good start, but you're not testing all cases. Look at the implementation and see what it does. Also make sure that an error is actually wrapped.

e.g.:

```go
	errBar := errors.New("bar")

	err := fmt.Errorf("foo, %w", errBar)
	errors.Is(err, errBar) // error "Foo" Wraps errBar, so this test should pass
```